src/test/scala/gatling/ReviewingAPie.scala

```diff
@@ -0,0 +1,55 @@
+package gatling
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import scala.util.Random
+
+object ReviewingAPie {
+  private val searchData = csv("gatling/data/search.csv")
+  private val reviewsData = csv("gatling/data/reviews.csv")
```

Comment:
You need .circular so when the feed goes through the data once the test doesnt stop

Reply:
so '.../search.csv").circular' like that?